.travis.yml

```diff
@@ -0,0 +1,16 @@
+---
+sudo: required
+git:
+  depth: 1
+services:
+  - docker
+pre_install:
+  - docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
```

Comment:
This build will fail, because I haven't figured out how to get the docker creds from apo